### DISCUSSION TITLE
Filtering out Recycle Bin, System Volume Information, etc.

### DISCUSSION THREAD

**Discussion by chinagreenelvis** [WinMerge/winmerge#1775] [Q&A]
Can I make a filter file that always filters out these directories on any drive, or do I have to specify every single drive I want them filtered out of?

**Comment by sdottaka** [comment]
A file filter like the following would ignore the $Recycle.Bin and System Volume Information folders on any drive.
For more information on file filters, please see:
https://manual.winmerge.org/en/Filters.html

~~~
name: #1775
desc: #1775 Longer description
def: include
d: \\\$Recycle\.bin$
d: \\System Volume Information$
~~~
